rllib/algorithms/appo/appo.py

```diff
@@ -193,6 +198,14 @@ def training(
                 on before updating the target networks and tune the kl loss
                 coefficients. NOTE: This parameter is only applicable when using the
                 Learner API (enable_rl_module_and_learner=True).
+            circular_buffer_num_batches: The number of train batches that fit
+                into the circular buffer. Each such train batch can be sampled for
+                training max. `circular_buffer_iterations_per_batch_K` times.
```

Comment:
The `circular_buffer_iterations_per_batch_K` is irritating as it is desribed nowhere. Instead we have a very similar attribute below.